docs/crud/queries.md

```diff
@@ -4,4 +4,242 @@ title: Queries
 sidebar_label: Queries
 ---
 
-## TODO
\ No newline at end of file
+Graphback provides two query types for every model in the schema.
+
+- [`find<Type>s`](#find): fetch all or a subset of items through optional filtering, pagination and sorting.
+- [`get<Type>`](#get): fetch a single item by its unique ID.
+
+## find
+
+The `find` operation allows the client to fetch multiple items from the database, with optional filtering, pagination and ordering of the data allowing you to specify exactly the data you need. 
+
+The query follows the naming format `find<Type>s`:
+
+```graphql
+type Query {
+  findNotes(filter: NoteFilter, page: PageRequest, orderBy: OrderByInput): NoteResultList!
+}
+```
+
+- `filter`: filter object to query specific data.
+- `page`: enables pagination of the data.
+- `orderBy`: sorts the data in ascending or descending order.
+
+The items fetched by `findNote` are returned in a container type `NoteResultList` created by Graphback:
+
+```graphql
+type NoteResultList {
+  # The notes are returned in the items field
+  items: [Note]!
+  # Offset specified in query
+  offset: Int
+  # Offset specified in query
+  limit: Int
+  # total count of notes in the database
+  count: Int
+}
+```
+
+Example of a simple query to retrieve all notes:
+
+```graphql
+findNotes {
+  items {
+    id
+    title
+    likes
+  }
+}
+```
+
+### Filtering
```

Comment:
So this docs are more detailed than our own https://graphqlcrud.org docs :)
Good job!